anomalib/utils/callbacks/visualizer_callback.py

```diff
@@ -61,17 +57,19 @@ def __init__(
         self.save_images = save_images
         self.image_save_path = Path(image_save_path)
 
```

Comment:
maybe add value check to `task` and `mode` here as well?

Reply:
I've added the checks, but it clutters the classes a bit. Maybe we can switch to using enums for selectable parameters in the future.

Reply:
enums would be good to avoid such cases. Their disadvantage, though, is it couples the code a bit.